app.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const { getCategories } = require('../be-nc-games/controllers/games.controllers');
+const app = express();
+
+app.get('/api/categories', getCategories)
+
+app.use((err, req, res, next) => {
```

Comment:
Nice that you've implemented an error handler, I'm not sure however that this would ever trigger as the status is only applied to the response. Essentially, remove the `if` statement here and you can stick a console.log inside this error handler for future reference